.github/workflows/check_wdk.yml

```
@@ -0,0 +1,62 @@
+    - name: Check for updates to the Windows Driver Kit
+      id: check_wdk
+      run: |
+        # Get the latest version of the Windows Driver Kit
+        $packages = nuget list "Microsoft.Windows.WDK.x64"
+        $packageLine = $package | Where-Object { $_ -match $packageName }
+        $packageVersion = $packageLine -replace "Microsoft.Windows.WDK.x64", ""
+        echo "::set-output name=wdk_version::$packageVersion"
+
+    - name: Check the version of the WDK in the repo
+      id: check_repo_wdk
+      run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)